packages/client/components/ActivityLibrary/ActivityDetailsSidebar.tsx

```diff
@@ -0,0 +1,125 @@
+import React, {useState} from 'react'
+import graphql from 'babel-plugin-relay/macro'
+import {useFragment} from 'react-relay'
+import StartRetrospectiveMutation from '~/mutations/StartRetrospectiveMutation'
+import {ActivityDetailsSidebar_template$key} from '~/__generated__/ActivityDetailsSidebar_template.graphql'
+import {ActivityDetailsSidebar_teams$key} from '~/__generated__/ActivityDetailsSidebar_teams.graphql'
+import NewMeetingTeamPicker from '../NewMeetingTeamPicker'
+import {MenuPosition} from '../../hooks/useCoords'
+import sortByTier from '../../utils/sortByTier'
+import NewMeetingSettingsToggleCheckIn from '../NewMeetingSettingsToggleCheckIn'
+import NewMeetingSettingsToggleAnonymity from '../NewMeetingSettingsToggleAnonymity'
+import NewMeetingActionsCurrentMeetings from '../NewMeetingActionsCurrentMeetings'
+import FlatPrimaryButton from '../FlatPrimaryButton'
+import SelectTemplateMutation from '../../mutations/SelectTemplateMutation'
+import useAtmosphere from '../../hooks/useAtmosphere'
+import useMutationProps from '../../hooks/useMutationProps'
+import {useHistory} from 'react-router'
+
+interface Props {
+  selectedTemplateRef: ActivityDetailsSidebar_template$key
+  teamsRef: ActivityDetailsSidebar_teams$key
+}
+
+const ActivityDetailsSidebar = (props: Props) => {
+  const {selectedTemplateRef, teamsRef} = props
+  const selectedTemplate = useFragment(
+    graphql`
+      fragment ActivityDetailsSidebar_template on MeetingTemplate {
+        id
+        teamId
+        orgId
+        scope
+      }
+    `,
+    selectedTemplateRef
+  )
+
+  const teams = useFragment(
+    graphql`
+      fragment ActivityDetailsSidebar_teams on Team @relay(plural: true) {
```

Comment:
Question: are the 2 fragments because of that weird relay bug with optimistic objects getting GC'd?

Reply:
Not at all, this is just the component API that makes the most sense here. Otherwise we'd be passing in a viewer ref + template ID and duplicating a lot of the logic in the parent `ActivityDetails` component.